vizro-core/docs/pages/user-guides/layouts.md

```diff
@@ -403,10 +410,278 @@ By default, the grid fits all charts/components on the screen. This can lead to
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                type: graph
+            layout:
+              type: flex
+            title: Flex - basic example
+        ```
+
+    === "Result"
+        [![FlexBasic]][flexbasic]
+
+### Flex - advanced example
+
+If you want to customize the default behavior, the [Flex][vizro.models.Flex] model allows you to configure three optional arguments:
+
+- `direction`: Defines the layout direction of the components within the flex container, determining whether they are arranged in rows or columns.
+- `gap`: Controls the spacing between components in the flex container, enabling you to set consistent horizontal and vertical spacing between items.
+- `wrap`: Determines whether components should wrap onto multiple lines or remain on a single line when there isn't enough space in the container.
+
+!!! example "Flex - advanced example"
+    === "app.py"
+        ```{.python pycafe-link}
+        import vizro.models as vm
+        from vizro import Vizro
+        import vizro.plotly.express as px
+
+        tips = px.data.tips()
+
+        page = vm.Page(
+            title="Flex - advanced example",
+            layout=vm.Flex(direction="row", gap="40px", wrap=True),
+            components=[vm.Graph(figure=px.violin(tips, y="tip", x="day", color="day")) for i in range(3)],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+        Vizro().build(dashboard).run()
+        ```
+
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to add data to the data manager and parse YAML configuration
+        # See yaml_version example
+        pages:
+          - components:
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                type: graph
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                type: graph
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                type: graph
+            layout:
+              type: flex
+              direction: row
+              gap: 40px
+              wrap: true
+            title: Flex - advanced example
+        ```
+
+    === "Result"
+        [![FlexAdvanced]][flexadvanced]
+
+### Change the size of flex-items
+
+There may be times when resizing your flex items is necessary, such as when designing for different screen sizes or combining elements with diverse content types (e.g., charts, text, or images).
+
+You can achieve this by either specifying the `width` and `height` directly on the components or, if you're comfortable, applying custom CSS.
```

Comment:
```suggestion
You can achieve this by either specifying the `width` and `height` directly on the components or by applying custom CSS.
```

Reply:
I'm uncomfortable with "comfortable" since it creates indecision in the reader, "Am I comfortable?" because it doesn't really say what the requirements are to be comfortable.

Reply:
You could also add anchors to the headings below for each option again, so the reader can see there is more guidance coming.